### DISCUSSION TITLE
No energy consumption reported by Aubess Zigbee Smart Switches

### DISCUSSION THREAD

**Discussion by zkvvoob** [zigpy/zha-device-handlers#2705] [Q&A]
Hello all,

I've bought a few of [these smart switches](https://www.aliexpress.com/item/1005005744193282.html) and have been using them just fine for the past couple of weeks. Even though they list a lot of sensors, none of the switches show energy consumption.

![71edc7f39ec8add0ce942250cc0e10ac3c8c402c](https://github.com/zigpy/zha-device-handlers/assets/7645808/967e786c-b689-4125-a8a1-c53cb26dc30e)

I'm using Zigbee Home Automation (ZHA) in Home Assistant. 

Do you know if a quirk is needed in order for the energy consumption values to start flowing in?

Thank you!

**Comment by Hedda** [comment]
> Do you know if a quirk is needed in order for the energy consumption values to start flowing in?

Probably, so do submit a new "Device support request" issue under Issues -> https://github.com/zigpy/zha-device-handlers/issues/

Edit ZHA docs if unclear -> https://www.home-assistant.io/integrations/zha#how-to-add-support-for-new-and-unsupported-devices
